### PR TITLE
Change logic for mobile, implement most rented book

### PR DESCRIPTION
**Pull request by DragisaDraskovic**
Last pull request

### CODE REVIEW

File: src/components/mostRentedBooks/MostRentedBooks.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react'
+
+import RentalServices from '../../services/RentalServices'
+import './MostRentedBooks.css'
+import { MostRented } from '../../model/MostRentedBooksResponse'
+import Card from '../card/Card'
+
+const MostRentedBooks = () => {
+  const [ mostRentedBooks, setMostRentedBooks ] = useState<MostRented[]>([])
+  console.log(mostRentedBooks)
```
Comment (vladimir-agbaba-valcon) [inline src/components/mostRentedBooks/MostRentedBooks.tsx]: console.log

File: src/services/RentalServices.tsx [hunk 2]
```diff
@@ -19,4 +19,9 @@ const getBookHistory = (bookId: number) => {
   return response
 }
 
-export default { rentBook, returnBook, getBookHistory }
+const getMostRented = (count : number) => {
+  const response = axios.get(process.env.REACT_APP_BASE_URL + `/api/Rental/top/${count}`,config)
```
Comment (vladimir-agbaba-valcon) [inline src/services/RentalServices.tsx:23]: Create const BASE_URL = `${process.env.REACT_APP_BASE_URL}/api/Rental`

File: src/components/mostRentedBooks/MostRentedBooks.tsx (hunk 1 above)
Comment (DragisaDraskovic) [reply src/components/mostRentedBooks/MostRentedBooks.tsx]: fix

File: src/services/RentalServices.tsx (hunk 2 above)
Comment (DragisaDraskovic) [reply src/services/RentalServices.tsx:23]: Ok. I will implement this.

**Review by vladimir-agbaba-valcon — APPROVED** [review]

**Pull request merged**